docs/modules/ROOT/pages/reference/extensions/crypto.adoc

```diff
@@ -45,6 +45,51 @@ ifeval::[{doc-show-user-guide-link} == true]
 Check the xref:user-guide/index.adoc[User guide] for more information about writing Camel Quarkus applications.
 endif::[]
 
+[id="extensions-crypto-usage"]
+== Usage
+[id="extensions-crypto-usage-fips"]
+=== FIPS
+
+When running `crypto` extension on FIPS enabled system the BC-FIPS has to be utilized.
+
+* Exclude all `bc*` dependencies. Example:
+```
+                    <exclusions>
```

Comment:
Can we strip the large bits of leading whitespace from these code snippets? If we leave it, then it might look a bit weird if the Camel website renders it as-is.

Reply:
fixed